packages/color-area/test/color-area.test.ts

```diff
@@ -325,6 +330,112 @@ describe('ColorArea', () => {
         expect(el.x).to.equal(0.53125);
         expect(el.y).to.equal(0.53125);
     });
+    it('responds to events on the internal input element', async () => {
+        const inputSpy = spy();
+        const changeSpy = spy();
+        const el = await fixture<ColorArea>(
+            html`
+                <sp-color-area
+                    color="hsla(100, 50%, 50%, 1)"
+                    @change=${() => changeSpy()}
+                    @input=${() => inputSpy()}
+                ></sp-color-area>
+            `
+        );
+
+        await elementUpdated(el);
+
+        el.inputX.focus();
+
+        el.inputX.dispatchEvent(
+            new Event('input', {
+                bubbles: true,
+                composed: true,
+            })
+        );
+        el.inputX.dispatchEvent(
+            new Event('change', {
+                bubbles: true,
+                composed: false, // native change events do not compose themselves by default
+            })
+        );
+
+        expect(inputSpy.callCount).to.equal(1);
+        expect(changeSpy.callCount).to.equal(1);
+
+        el.inputY.focus();
+
+        el.inputY.dispatchEvent(
+            new Event('input', {
+                bubbles: true,
+                composed: true,
+            })
+        );
+        el.inputY.dispatchEvent(
+            new Event('change', {
+                bubbles: true,
+                composed: false, // native change events do not compose themselves by default
+            })
+        );
+
+        expect(inputSpy.callCount).to.equal(2);
+        expect(changeSpy.callCount).to.equal(2);
+    });
+    it('dispatches input and change events in response to "Arrow*" keypresses', async () => {
+        const inputSpy = spy();
+        const changeSpy = spy();
+        const el = await fixture<ColorArea>(
+            html`
+                <sp-color-area
+                    color="hsla(100, 50%, 50%, 1)"
+                    @change=${() => changeSpy()}
+                    @input=${() => inputSpy()}
+                ></sp-color-area>
+            `
+        );
+
+        await elementUpdated(el);
+
+        el.inputX.focus();
+
+        await sendKeys({ press: 'ArrowRight' });
+        await sendKeys({ press: 'ArrowRight' });
+
+        await elementUpdated(el);
+
+        expect(inputSpy.callCount).to.equal(4);
+        expect(changeSpy.callCount).to.equal(2);
```

Comment:
These should match, I think. One input and one change event for each key listened to... can you check what's causing this? Seems like maybe the `event.preventDefault()` note I sent before, but possibly something else.

Reply:
For reference: https://github.com/adobe/spectrum-web-components/blob/main/packages/color-slider/test/color-slider.test.ts#L80-L95